trojsten/submit/templatetags/submit_parts.py

```diff
@@ -11,12 +11,13 @@
 
 
 @register.inclusion_tag("trojsten/submit/parts/submit_form.html", takes_context=True)
-def show_submit_form(context, task, user, redirect):
+def show_submit_form(context, task, user, redirect, source_only=False):
```

Comment:
Co znamena _source_only_ v tomto kontexte?  Vies tomu dat viac vravnejsie meno?
V tych templatoch `task.has_descripion and not source_only` znie ako keby ta druha cast bola zbytocna.

Reply:
task.has_description pochádza z modelu daného tasku. source_only skryje zoznam a formulár na odovzdávanie popisov, lebo to na stránke submitu nie je až také podstatné.